megaqc/database.py

```diff
@@ -92,17 +94,51 @@ def init_db(url):
     """ Initialise a new database """
     if "postgresql" in url:
         try:
-            create_engine(url).connect().close()
-        except:
+            # lazy load for test envs without psycopg2 installed
+            from psycopg2.errors import DuplicateObject
```

Comment:
I moved it in there because `psycopg2-binary` is not installed by default (including in the CI test runner). Not sure if this is fixed in your setup update PR